File: python/pangram/pangram.py
def is_pangram(sentence):
    """
    Determine if a sentence is a pangram.
    A pangram is a sentence using every letter of the alphabet at least once.
    
    :param sentence: The input sentence string
    :return: True if sentence is a pangram and False if it is not
    """
    return len(list(filter(lambda x: x.isalpha(), set(sentence)))) == 26

File: typescript/pangram/pangram.ts
export default class Pangram {
    private sentence: string = ''

    constructor(sentence: string) {
        this.sentence = sentence
    }

    isPangram(): boolean {
        return this.sentence.toLowerCase().split('').filter((val, index, self) => {
            return self.indexOf(val) === index && /^[a-zA-Z]+$/.test(val)
        }).length === 26
    }
}